### PR TITLE
Print informative message after "lago ovirt start"

### PR DESCRIPTION
**Pull request by gbenhaim**
The message will include the url to the web UI, username and password.

Signed-off-by: gbenhaim <galbh2@gmail.com>

### CODE REVIEW

File: ovirtlago/cmd.py [hunk 1]
```diff
@@ -240,6 +241,19 @@ def do_ovirt_start(prefix, with_vms, **kwargs):
             with LogTask('Starting Engine VMs'):
                 prefix.virt_env.engine_vm().start_all_vms()
 
+    LOGGER.info(
+        '\n'
+        'oVirt environment is ready to be used.\n'
+        'You can access the web UI with the following link and credentials:\n'
+        'https://{ip}\n'
+        'Username: {username}\n'
+        'Password: {password}'.format(
```
Comment (nvgoldin) [inline ovirtlago/cmd.py]: I think using """ would be nicer(and then you don't need the explicit eol'), i.e.:
```python
LOGGER.info("""

oVirt ....

""".format)
```

**Comment by nvgoldin** [comment]
great idea!
maybe add it to 'lago ovirt status' as well

**Comment by gbenhaim** [comment]
@nvgoldin @mykaul 
Other thoughts before I merge ?

**Comment by gbenhaim** [comment]
ci test please

**Comment by gbenhaim** [comment]
ci test please

**Comment by gbenhaim** [comment]
ci merge please

**Pull request merged**